drivers/staging/vc04_services/bcm2835-isp/bcm2835-v4l2-isp.c

```diff
@@ -1436,10 +1444,12 @@ static int register_node(struct bcm2835_isp_dev *dev,
 	}
 
 	/* Define the device names */
-	snprintf(vfd->name, sizeof(node->vfd.name), "%s-%s%d", BCM2835_ISP_NAME,
-		 node->name, node->id);
+	snprintf(vfd->name, sizeof(node->vfd.name), "%s%d-%s%d",
+		 BCM2835_ISP_NAME, instance, node->name, node->id);
 
-	ret = video_register_device(vfd, VFL_TYPE_VIDEO, video_nr + index);
+	ret = video_register_device(vfd, VFL_TYPE_VIDEO,
+				    video_nr + index +
+				    (BCM2835_ISP_NUM_NODES * instance));
```

Comment:
Please can we have an array of base values as the module parameter.

Deinterlace is already registering as 18, and HEVC decode as 19.

Reply:
Ack.